app/uk/gov/hmrc/pdsauthcheckerapi/config/AppConfig.scala

```diff
@@ -18,9 +18,16 @@ package uk.gov.hmrc.pdsauthcheckerapi.config
 
 import javax.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.pdsauthcheckerapi.config.UKIMSServicesConfig
+import io.lemonlabs.uri.Url
 
 @Singleton
-class AppConfig @Inject()(config: Configuration) {
+class AppConfig @Inject() (
+    config: Configuration,
+    servicesConfig: UKIMSServicesConfig
+) {
 
   val appName: String = config.get[String]("appName")
+
+  val pdsStubAuthCheckerUrl = Url.parse(servicesConfig.baseUrl("eis-stub"))
```

Comment:
you're gonna be frustrated, but 1) lets fix the name of the val
2) eis-stub should probably be eis or cau01 since that's what we're targetting in prod, not the stub.